src/analysis/normed_space/bounded_linear_maps.lean

```diff
@@ -277,22 +253,28 @@ def is_bounded_bilinear_map.deriv (h : is_bounded_bilinear_map k f) (p : E × F)
   smul := λc q, begin
     change f (p.1, c • q.2) + f (c • q.1, p.2) = c • (f (p.1, q.2) + f (q.1, p.2)),
     simp [h.smul_left, h.smul_right, smul_add]
-  end,
-  bound := begin
-    rcases h.bound with ⟨C, Cpos, hC⟩,
-    refine exists_pos_bound_of_bound (C * ∥p.1∥ + C * ∥p.2∥) (λq, _),
-    calc ∥f (p.1, q.2) + f (q.1, p.2)∥
-      ≤ ∥f (p.1, q.2)∥ + ∥f (q.1, p.2)∥ : norm_triangle _ _
-    ... ≤ C * ∥p.1∥ * ∥q.2∥ + C * ∥q.1∥ * ∥p.2∥ : add_le_add (hC _ _) (hC _ _)
-    ... ≤ C * ∥p.1∥ * ∥q∥ + C * ∥q∥ * ∥p.2∥ : begin
-       apply add_le_add,
-       exact mul_le_mul_of_nonneg_left (le_max_right _ _) (mul_nonneg (le_of_lt Cpos) (norm_nonneg _)),
-       apply mul_le_mul_of_nonneg_right _ (norm_nonneg _),
-       exact mul_le_mul_of_nonneg_left (le_max_left _ _) (le_of_lt Cpos),
-    end
-    ... = (C * ∥p.1∥ + C * ∥p.2∥) * ∥q∥ : by ring
   end }
 
+def is_bounded_bilinear_map.deriv (h : is_bounded_bilinear_map k f) (p : E × F) : (E × F) →L[k] G :=
+(h.linear_deriv p).with_bound $ begin
+  rcases h.bound with ⟨C, Cpos, hC⟩,
+  refine ⟨C * ∥p.1∥ + C * ∥p.2∥, λq, _⟩,
+  calc ∥f (p.1, q.2) + f (q.1, p.2)∥
+    ≤ ∥f (p.1, q.2)∥ + ∥f (q.1, p.2)∥ : norm_triangle _ _
+  ... ≤ C * ∥p.1∥ * ∥q.2∥ + C * ∥q.1∥ * ∥p.2∥ : add_le_add (hC _ _) (hC _ _)
+  ... ≤ C * ∥p.1∥ * ∥q∥ + C * ∥q∥ * ∥p.2∥ : begin
+      apply add_le_add,
+      exact mul_le_mul_of_nonneg_left (le_max_right _ _) (mul_nonneg (le_of_lt Cpos) (norm_nonneg _)),
+      apply mul_le_mul_of_nonneg_right _ (norm_nonneg _),
+      exact mul_le_mul_of_nonneg_left (le_max_left _ _) (le_of_lt Cpos),
+  end
+  ... = (C * ∥p.1∥ + C * ∥p.2∥) * ∥q∥ : by ring
+end
+
+set_option class.instance_max_depth 150
```

Comment:
@digama0 Should we worry about numbers this big? Or is this not a problem at all?